android/app/src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,7 @@
     <activity
       android:name=".MainActivity"
       android:label="@string/app_name"
+      android:screenOrientation="portrait"
```

Comment:
Can we put comments in here? Since this is a template file that comes from React Native, we could easily delete this line during an upgrade. So, we need to say "Edge addition: Fix for resolution issue" or something like that.